app/src/main/java/org/dhis2/usescases/main/program/HomeRepositoryImpl.kt

```diff
@@ -10,360 +10,383 @@ import org.hisp.dhis.android.core.dataset.DataSet
 import org.hisp.dhis.android.core.dataset.DataSetCollectionRepository
 import org.hisp.dhis.android.core.enrollment.Enrollment
 import org.hisp.dhis.android.core.enrollment.EnrollmentStatus
+import org.hisp.dhis.android.core.event.EventStatus
 import org.hisp.dhis.android.core.organisationunit.OrganisationUnit
 import org.hisp.dhis.android.core.period.DatePeriod
 import org.hisp.dhis.android.core.program.ProgramCollectionRepository
 import org.hisp.dhis.android.core.program.ProgramType.WITHOUT_REGISTRATION
 import org.hisp.dhis.android.core.program.ProgramType.WITH_REGISTRATION
+import java.util.*
+import kotlin.collections.ArrayList
 
 internal class HomeRepositoryImpl(private val d2: D2, private val eventLabel: String) :
```

Comment:
KtlintFormat was not execute in this file. imports are not in lexicographical order.